packages/host/app/components/operator-mode/container.gts

```diff
@@ -136,6 +139,65 @@ export default class OperatorModeContainer extends Component<Signature> {
     return await this.operatorModeStateService.constructRecentCards();
   });
 
+  @action async onCommand(command: any) {
+    // apply patch
+    if (command.type == 'patch') {
+      for (let item of this.allStackItems) {
+        if ('card' in item && item.card.id == command.id) {
+          await this.setFieldValues(item.card, command.patch.attributes);
+          return await this.save(item);
+        }
+      }
+    }
+  }
+
+  @action async onPreviewCommand(command: any) {
+    // apply patch
+    if (command.type == 'patch') {
```

Comment:
i'm not fully convinced this is actually going to preview--i think there are scenarios where this will actually perform a save on the card